version_control_pattern.md

```diff
@@ -20,16 +20,24 @@ git checkout -b my-new-sensibly-named-branch
 ```
 3. Make some commits on the new branch.
 ```sh
-# Make some changes then stage them all.
-git add .
-# Or stage a specific file.
-git add README.md
+# Make some changes then stage each file you've changed - e.g. file1.txt and file2.txt.
```

Comment:
This _is_ the better option but may not be a realistic expectation.  Also, using `git commit` and entering the commit message in a test editor, rather than `git commit -m "my awesome commit message"` allows the author to preview everything that has been staged before committing to the commit, as it were.

Perhaps we should present both options but state a preference for one (the one you have added)?